examples/actix-web.rs

```diff
@@ -0,0 +1,103 @@
+use std::sync::Mutex;
+
+use actix_web::{web, App, HttpResponse, HttpServer, Responder, Result};
+use prometheus_client::encoding::text::{encode, Encode};
+use prometheus_client::metrics::counter::Counter;
+use prometheus_client::metrics::family::Family;
+use prometheus_client::registry::Registry;
+
+#[derive(Clone, Hash, PartialEq, Eq, Encode)]
+pub enum RequestType {
```

Comment:
```suggestion
pub enum Method {
```

In the `tide` example we call it `Method`. I don't really care about the names, though I would prefer to be consistent. What do you think?

https://github.com/prometheus/client_rust/blob/master/examples/tide.rs

Reply:
Good point, agree.